standalone-metastore/metastore-common/src/gen/thrift/gen-javabean/org/apache/hadoop/hive/metastore/api/GetFileMetadataRequest.java

```diff
@@ -396,13 +396,13 @@ public void write(org.apache.thrift.protocol.TProtocol prot, GetFileMetadataRequ
     public void read(org.apache.thrift.protocol.TProtocol prot, GetFileMetadataRequest struct) throws org.apache.thrift.TException {
       org.apache.thrift.protocol.TTupleProtocol iprot = (org.apache.thrift.protocol.TTupleProtocol) prot;
       {
-        org.apache.thrift.protocol.TList _list1017 = iprot.readListBegin(org.apache.thrift.protocol.TType.I64);
-        struct.fileIds = new java.util.ArrayList<java.lang.Long>(_list1017.size);
-        long _elem1018;
-        for (int _i1019 = 0; _i1019 < _list1017.size; ++_i1019)
+        org.apache.thrift.protocol.TList _list1027 = iprot.readListBegin(org.apache.thrift.protocol.TType.I64);
```

Comment:
nit: do you know why changes happen here? given that we only change `FireEventRequestData` in the `hive_metastore.thrift`.

Reply:
These changes occur with the change in the FireEventRequest object. I did a lot of tests around this on why so many thrift files are changing when I changed the FireEventRequest object but I couldn't figure out a reason.

Reply:
The result shows no diff when I complied the thrift with this PR.